src/main.py

```
@@ -0,0 +1,6 @@
+def main():
```

Ruff (0.12.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)